### ISSUE TITLE
Interface "Symplify\RuleDocGenerator\Contract\DocumentedRuleInterface" not found

### ISSUE THREAD

**Issue by lptn** [rectorphp/rector#5909]
# Bug Report

| Subject        | Details                                                         |
| :------------- | :---------------------------------------------------------------|
| Rector version | v0.9.31           |
| Installed as   | composer dependency                            |

After composer update I've got that error:

```sh
./vendor/bin/rector process --dry-run
```

```
[ERROR] Interface "Symplify\RuleDocGenerator\Contract\DocumentedRuleInterface" not found                                             
```

Updated composer packages:
```
 - Upgrading phpstan/phpstan (0.12.81 => 0.12.82)
  - Upgrading symplify/astral (v9.2.11 => v9.2.12)
  - Upgrading symplify/autowire-array-parameter (v9.2.11 => v9.2.12)
  - Upgrading symplify/composer-json-manipulator (v9.2.11 => v9.2.12)
  - Upgrading symplify/console-color-diff (v9.2.11 => v9.2.12)
  - Upgrading symplify/console-package-builder (v9.2.11 => v9.2.12)
  - Upgrading symplify/easy-testing (v9.2.11 => v9.2.12)
  - Upgrading symplify/markdown-diff (v9.2.11 => v9.2.12)
  - Upgrading symplify/package-builder (v9.2.11 => v9.2.12)
  - Upgrading symplify/php-config-printer (v9.2.11 => v9.2.12)
  - Upgrading symplify/rule-doc-generator (v9.2.11 => v9.2.12)
  - Upgrading symplify/set-config-resolver (v9.2.11 => v9.2.12)
  - Upgrading symplify/skipper (v9.2.11 => v9.2.12)
  - Upgrading symplify/smart-file-system (v9.2.11 => v9.2.12)
  - Upgrading symplify/symplify-kernel (v9.2.11 => v9.2.12)
  

 ```

**Comment by TomasVotruba** [comment]
I'll release big fix 0.9 tag with this dependency

**Comment by TomasVotruba** [comment]
Fixed in https://github.com/rectorphp/rector/releases/tag/0.9.32

@lptn Thank you for reporting
